package.json

```diff
@@ -38,5 +38,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "jest": "^28.1.1"
```

Comment:
I don't think this is required. I removed it and (after rerunning npm install) the tests still pass.